deploy/helm/sumologic/templates/_helpers.tpl

```diff
@@ -778,3 +778,15 @@ Example:
               key: {{ template "terraform.sources.config-map-variable" (dict "Type" $type "Context" $ctx "Name" $key) }}
 {{- end }}
 {{- end -}}
+
+{{/*
+Generate a space separated list of quoted values:
+
+Example:
+
+{{ include "helm-toolkit.utils.joinListWithSpaces" .Values.sumologic.logs.fields }}
+*/}}
+{{- define "helm-toolkit.utils.joinListWithSpaces" -}}
+{{- $local := dict "first" true -}}
+{{- range $k, $v := . -}}{{- if not $local.first }} {{ end -}}{{ $v | quote }}{{- $_ := set $local "first" false -}}{{- end -}}
```

Comment:
could you introduce string variable and split this concatenation on multiple lines?